gemini/function-calling/sql-talk-app/app.py

```diff
@@ -129,7 +140,7 @@
 
     with st.chat_message("assistant"):
         message_placeholder = st.empty()
-        full_response = ""
+        FULL_RESPONSE = ""
```

Comment:
Why was this changed? `FULL_RESPONSE` and `BACKEND_DETAILS` aren't consts from what I can see.

Reply:
The linter is complaining to me about them and I'm not sure why 🤔

Reply:
I think the issue wasn't the upper case, but the linter just didn't recognize that some of these values were edited?

Reply:
Oops, I was confusing those with some other vars it was complaining about. I think here it is complaining because I'm setting it to be what it thinks is a constant (empty) string, which I have to do because of Streamlit's reactivity. I could add `#noQA` tags to these instead of uppercasing.

Reply:
I don't think any linter will pick up that it will be edited later, it's probably just looking at whether the right hand side is a variable or calculated value vs. a certain data type (str, int). I'm okay adding `# pylint: disable=` to these lines if that works.